packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -221,7 +221,7 @@ type MenuItemProps = {
 };
 const MenuItem = (props: MenuItemProps) => (
   <li
-    role="menu-item"
+    role="link"
```

Comment:
Unrelated, eslint a11y was complaining.

Reply:
Do you remember why we don't put a regular `a` into the `li`?

Reply:
We do, it's a bit nested.

Reply:
Gotcha. Didn't expand the diff enough then :)